### PR TITLE
SEO-172844-WPF-weblink-validatetor-image

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Please review the changes made in image name alt text and image folder.
Thanks.
Regards,
Edith.

### CODE REVIEW

**Review by MallikaRK — APPROVED** [review]
Approved

File: wpf/DataGrid/Styles-and-Templates.md [hunk 1]
```diff
@@ -1435,4 +1435,4 @@ void sfDataGrid_ItemsSourceChanged(object sender, GridItemsSourceChangedEventArg
 {% endhighlight %}
 {% endtabs %}
 
-![WPF DataGrid displays Busy Indicator before Loading Data](Styles-and-Templates_images/wpf-datagrid-busy-indicator.png)
+![WPF DataGrid displays Busy Indicator before Loading Data](styles-and-templates_images/wpf-datagrid-busy-indicator.png)
```
Comment (Girija-Rajendran) [inline wpf/DataGrid/Styles-and-Templates.md:1438]: Make sure it has changed in the folder.

Comment (Edithabich) [reply wpf/DataGrid/Styles-and-Templates.md:1438]: corrected

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**